### ISSUE TITLE
Accessibility-Tooltip Content get disoriented and outside after applying Text Spacing

### ISSUE THREAD

**Issue by LucyLiYY** [highcharts/highcharts#18323]
#### Expected behaviour
Tooltip content should display oriented inside after Text Spacing is enabled.

#### Actual behaviour
Tooltip Content get disoriented and outside after applying Text Spacing

#### Live demo with steps to reproduce

https://www.highcharts.com/demo/column-stacked
https://www.highcharts.com/demo/bar-stacked
![image](https://user-images.githubusercontent.com/17995942/213309588-9b4d71e8-7613-4cc0-afae-4457e4d78128.png)

#### Product version

Latest version

#### Affected browser(s)
Chrome, Edge

**Comment by karolkolodziej** [comment]
Hi @LucyLiYY 
Could you please let me know how you apply this Text Spacing? 
I'm asking because I'm not able to recreate that - [demo](https://jsfiddle.net/BlackLabel/0qbjy732/).

**Comment by msftedad** [comment]
Hi @karolkolodziej, Text spacing is applied using text spacing tool-stylus.
